app/controllers/async_info_controller.rb

```diff
@@ -53,25 +53,27 @@ def broadcast_data
   def user_data
     Rails.cache.fetch(user_cache_key, expires_in: 15.minutes) do
       {
-        id: @user.id,
-        name: @user.name,
-        username: @user.username,
-        profile_image_90: ProfileImage.new(@user).get(width: 90),
-        followed_tags: @user.cached_followed_tags.to_json(only: %i[id name bg_color_hex text_color_hex hotness_score], methods: [:points]),
-        followed_user_ids: @user.cached_following_users_ids,
-        followed_podcast_ids: @user.cached_following_podcasts_ids,
-        reading_list_ids: ReadingList.new(@user).cached_ids_of_articles,
         blocked_user_ids: @user.all_blocking.pluck(:blocked_id),
-        saw_onboarding: @user.saw_onboarding,
         checked_code_of_conduct: @user.checked_code_of_conduct,
         checked_terms_and_conditions: @user.checked_terms_and_conditions,
-        display_sponsors: @user.display_sponsors,
+        config_body_class: @user.config_body_class,
+        created_at: @user.created_at,
         display_announcements: @user.display_announcements,
-        trusted: @user.trusted,
+        display_sponsors: @user.display_sponsors,
+        email: @user.email,
+        followed_podcast_ids: @user.cached_following_podcasts_ids,
+        followed_tags: @user.cached_followed_tags.to_json(only: %i[id name bg_color_hex text_color_hex hotness_score], methods: [:points]),
+        followed_user_ids: @user.cached_following_users_ids,
+        id: @user.id,
         moderator_for_tags: @user.moderator_for_tags,
-        config_body_class: @user.config_body_class,
+        name: @user.name,
         pro: @user.pro?,
-        created_at: @user.created_at
+        profile_image_90: ProfileImage.new(@user).get(width: 90),
+        reading_list_ids: ReadingList.new(@user).cached_ids_of_articles,
+        saw_onboarding: @user.saw_onboarding,
+        subscription_source_article_ids: @user.cached_subscription_source_article_ids,
```

Comment:
I alphabetized this list and added:
```ruby
email: @user.email
subscription_source_article_ids: @user.cached_subscription_source_article_ids
```
We want the `email` to populate text in the liquid tag. Something like:
> Hey, user! Are you sure you want to share your email address (test@test.com) with the author_username?

The exact wording is TBD, but we'll want the email address so it's as clear as possible to the user what they're agreeing to.

And we want `subscription_source_article_ids` so we know which Authors the current user is subscribed to and through which articles. Then we know to display the "already subscribed" UX.